src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWarningsFilter.java

```diff
@@ -25,8 +25,90 @@
 import com.puppycrawl.tools.checkstyle.checks.SuppressWarningsHolder;
 
 /**
- * A filter that suppresses warnings using the {@link SuppressWarnings}
- * annotation.
+ * <p>
+ * Filter {@code SuppressWarningsFilter} uses annotations to suppress audit events.
```

Comment:
> uses annotations 

The original used `the {@link SuppressWarnings} annotation` as we don't allow any other annotation but `SuppressWarnings`. The current new wording may make it sound like we would accept other annotations in my mind.
We do specify the `SuppressWarnings` in the next paragraph but this first one seems the most important that we should mention it.

Reply:
done